pom.xml

```diff
@@ -114,6 +114,9 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.20.1</version>
+                    <configuration>
+                        <argLine>-Djenkins.test.timeout=480</argLine>
```

Comment:
Does it make sense to take a granular approach using [JUnit's built-in test timeout feature](https://github.com/junit-team/junit4/wiki/timeout-for-tests)?
I'm concerned that setting a timeout for all tests has the potential to cover up issues with other tests taking longer than they should.

Reply:
Tests that use JenkinsRule won't be affected by the test timeout from JUnit, and this timeout would only apply in tests that use JenkinsRule.

Reply:
Good change.. sometimes I had that issue too...

Reply:
Updated